src/main/java/com/mealmentor/enterprise/MealMentorController.java

```diff
@@ -50,8 +53,9 @@ public MealItem createMealItem(@RequestBody MealItem mealItem) throws Exception
         try{
             newMealItem = mealPlanService.save(mealItem);
         } catch (Exception e) {
-            //TODO ADD LOGGING
-
+            log.error("unable to save item", e);
+            e.printStackTrace();
+            throw(e);
```

Comment:
This was left as a todo. I added a logger and logged the exception and then threw the exception as you had the method expecting an exception to be thrown.

Reply:
Ah, great error handling.
Remove the e.printStackTrace().  log.error() already covers that, in a much better way.